ExpenseMaster.Tests/Repository/BudgetRepositoryTests.cs

```diff
@@ -0,0 +1,108 @@
+﻿using ExpenseMaster.DAL.DatabaseContext;
+using ExpenseMaster.DAL.Models;
+using ExpenseMaster.DAL.Repository;
+using Microsoft.EntityFrameworkCore;
+using Moq;
+using Xunit;
+
+namespace ExpenseMaster.Tests.Repository
+{
+    public class BudgetRepositoryTests
+    {
+        [Fact]
+        public async Task GetByIdAsync_ReturnsBudget()
```

Comment:
And we talk about three parts in name of test

Reply:
it's not my code and it's old